target/linux/ath79/image/generic.mk

```diff
@@ -724,6 +724,17 @@ define Device/comfast_cf-e375ac
 endef
 TARGET_DEVICES += comfast_cf-e375ac
 
+define Device/comfast_cf-e380ac-v2
+  SOC := qca9558
+  DEVICE_VENDOR := COMFAST
+  DEVICE_MODEL := CF-E380AC
+  DEVICE_VARIANT := v2
+  DEVICE_PACKAGES := kmod-usb-core kmod-usb2 \
+  kmod-ath10k-ct ath10k-firmware--ct
```

Comment:
This looks wrong: `ath10k-firmware--ct` I think you have to provide here the specific FW name you want to use

Reply:
this device has a QCA9880 chipset on it, this is probably a typo